src/types/skeleton.ts

```diff
@@ -0,0 +1,21 @@
+import { AnyElementOf } from '@/types/utils'
+
+export const SkeletonTypeArray = [
+  'table',
+  'card',
+  'form',
+  'spinner',
+  'fullscreen-kong',
+  'fullscreen-generic',
+  '', // default Skeleton.vue
+] as const
+
+export type SkeletonType = AnyElementOf<typeof SkeletonTypeArray>
+
+export const SkeletonBoxWidthArray = ['1', '2', '5', '6', '10', '25', '50', '75', '100', ''] as const
```

Comment:
Shouldn't the `SkeletonBoxWidthArray` NOT have empty string `''`? What is the implication when `SkeletonBoxWidth` is `''`?

Reply:
Same question, I don't think we should support `''` as a value, we don't handle it.

Reply:
`''` was used in components for `width`, changed it